src/components/Settings/LLMSettings/DefaultLLMSelector.tsx

```diff
@@ -11,6 +12,20 @@ interface DefaultLLMSelectorProps {
 
 const DefaultLLMSelector: React.FC<DefaultLLMSelectorProps> = ({ llmConfigs, defaultLLM, setDefaultLLM }) => {
   const [selectedLLM, setSelectedLLM] = useState(defaultLLM)
+  const [availableOllamaModels, setAvailableOllamaModels] = useState<LLMConfig[]>([])
+
+  useEffect(() => {
+    const fetchModels = async () => {
+      try {
+        const models = await window.llm.getAvailableModels()
```

Comment:
why do we need this call at all?

imo we should use the ```llmConfigs``` prop here and filter for the llms which have the ollama api interface: 
```  apiInterface: APIInterface```

Reply:
Resolved!